exporter/datadogexporter/translate_traces.go

```diff
@@ -51,6 +51,7 @@ const (
 	eventNameTag        string = "name"
 	eventAttrTag        string = "attributes"
 	eventTimeTag        string = "time"
+	MaxMetaValLen       int    = 5000
```

Comment:
Can you add the URL on the PR description here for reference?

Reply:
makes sense will do